tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -65,20 +65,11 @@ var _ = Describe("odo devfile push command tests", func() {
 			Expect(output).To(ContainSubstring("Executing devfile commands for component " + name))
 		})
 
-		It("should be able to push based on name passed", func() {
```

Comment:
Why are we removing this spec?

Reply:
because the check that this test is doing is done in the test bellow it https://github.com/openshift/odo/pull/3835/files#diff-c127bd883f89dd3a7abff6183ad9921dR87

Reply:
This is completely different scenario as compared to the one you mentioned. The spec you removed has a `name` field passing along with `odo push` command, which is nowhere verified throughout the spec AFAIK.

Can you please revert it back or adjust with your modified steps (may be it will be difficult).

Reply:
It’s not completely different, I will add the name in odo push

Reply:
Either way works for me.